.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -10,3 +10,6 @@ assignees: ''
 ### The problem faced currently?
 
 ### How can we solve it?
+
+>If you want this feature to be implemented, give it a thumbs up, so that we can determine which features are important to you.
```

Comment:
```suggestion
>If you want this feature to be implemented, give it a thumbs up reaction, so that we can determine which features are important to you.
```